lib/membrane_rtc_engine/engine.ex

```diff
@@ -566,18 +581,21 @@ defmodule Membrane.RTC.Engine do
     end)
   end
 
-  defp link_outbound_tracks(tracks, endpoint_id, ctx, state) do
+  defp link_outbound_tracks(tracks, endpoint_id, ctx) do
     Enum.reduce(tracks, {[], MapSet.new()}, fn
-      {track_id, encoding}, {new_links, not_linked} ->
-        tee = find_child(ctx, pattern: {:tee, {_other_endpoint_id, ^track_id}})
+      {track_id, format}, {new_links, not_linked} ->
+        tee =
```

Comment:
Maybe let's call this `format_specific_tee` to be consistent with the function above